namespaces/ci_job.yml

```diff
@@ -16,12 +16,12 @@ namespace:
     description: >
       Name of the CI job.
 
-  - name: build_id
+  - name: number
```

Comment:
`build_id` has different meaning in Jenkins, we want `number` here to make it unique and integer.